lib/import-export-visitor.js

```diff
@@ -347,11 +394,21 @@ module.exports = class ImportExportVisitor extends Visitor {
       !! getOption(options, "generateLetDeclarations");
     this.modifyAST = !! getOption(options, "ast");
     this.parse = getOption(options, "parse");
+    this.enforceStrictMode = getOption(options, "enforceStrictMode");
     this.nextKey = 0;
+    this.madeChanges = false;
 
     return this;
   }
 
+  visitProgram(path) {
+    this.visitChildren(path);
+    path.call(
+      firstStmtPath => this.getBlockBodyInfo(firstStmtPath),
+      "body", 0
+    );
+  }
```

Comment:
It makes sure `getBlockBodyInfo` gets called for the `Program`, even if there are no `import` or `export` declarations at the top level.